aulas/05.md

```diff
@@ -131,15 +131,15 @@ def create_access_token(data: dict):
     return encoded_jwt
 ```
 
-A função `create_access_token` é responsável por criar um novo token JWT que será usado para autenticar o usuário. Ela recebe um dicionário de dados, adiciona um tempo de expiração ao token (baseado na constante `ACCESS_TOKEN_EXPIRE_MINUTES`). Esses dados, em conjunto formam o **playload** do JWT. Em seguida usa a biblioteca `jose` para codificar essas informações em um token JWT, que é então retornado.
+A função `create_access_token` é responsável por criar um novo token JWT que será usado para autenticar o usuário. Ela recebe um dicionário de dados, adiciona um tempo de expiração ao token (baseado na constante `ACCESS_TOKEN_EXPIRE_MINUTES`). Esses dados, em conjunto, formam o **payload** do JWT. Em seguida, usa a biblioteca `jose` para codificar essas informações em um token JWT, que é então retornado.
 
 Note que a constante `SECRET_KEY` é usada para assinar o token, e o algoritmo `HS256` é usado para a codificação. Em um cenário de produção, você deve manter a `SECRET_KEY` em um local seguro e não expô-la em seu código.
 
 ### Testando a geração de tokens
 
-Embora esse código será coberto no futuro com a utilização do token, é interessante criarmos um teste para essa função com uma finalidade puramente didática. De forma em que consigamos ver os tokens gerados pelo `jose` e interagirmos com ele.
+Embora esse código será coberto no futuro com a utilização do token, é interessante criarmos um teste para essa função com uma finalidade puramente didática. De forma em que vejamos os tokens gerados pelo `jose` e interagirmos com ele.
```

Comment:
Eu dei uma pesquisada e minha sugestão é retirar o "em" do que tem no início da frase:
**OLD:** "De forma *em* que"
**NEW:** "De forma que"

Reply:
:heart: